docs/api/fiber.md

```diff
@@ -108,6 +108,7 @@ app.Listen(":8080", fiber.ListenConfig{
 | <Reference id="enableprefork">EnablePrefork</Reference>                 | `bool`                        | When set to true, this will spawn multiple Go processes listening on the same port.                                                           | `false` |
 | <Reference id="enableprintroutes">EnablePrintRoutes</Reference>         | `bool`                        | If set to true, will print all routes with their method, path, and handler.                                                                   | `false` |
 | <Reference id="gracefulcontext">GracefulContext</Reference>             | `context.Context`             | Field to shutdown Fiber by given context gracefully.                                                                                          | `nil`   |
+| <Reference id="gracefulshutdowntimeout">GracefulShutdownTimeout</Reference>| `time.Duration`            | Specifies the maximum duration to wait for the server to gracefully shutdown. When the timeout is reached, the shutdown process is interrupted and a `context.DeadlineExceeded` error is passed to the `OnShutdownError` callback. Set to 0 (default) to disable the timeout and wait indefinitely. | `0`   |
```

Comment:
I think we better write something like `graceful shutdown process is interrupted and it's forcibly` instead of `shutdown process interrupted`

Reply:
Can I change the description to the below?

Specifies the maximum duration to wait for the server to gracefully shutdown. When the timeout is reached, <mark>the graceful shutdown process is interrupted and forcibly terminated,</mark> and the `context.DeadlineExceeded` error is passed to the `OnShutdownError` callback. Set to 0 (default) to disable the timeout and wait indefinitely.

Reply:
> @efectn Should I change the description to the below?
> 
> > Specifies the maximum duration to wait for the server to gracefully shutdown. When the timeout is reached, the graceful shutdown process is interrupted and forcibly terminated, and the `context.DeadlineExceeded` error is passed to the `OnShutdownError` callback. Set to 0 (default) to disable the timeout and wait indefinitely.

Yes, i think it is OK